api/params.go

```diff
@@ -728,6 +735,12 @@ func (p *Params) SetDefaults(gitSource, gitOwner, gitName, appLabel, buildVersio
 			p.StorageMountPath = "/data"
 		}
 	}
+
+	// default dns annotations added
+	if !p.DNSParams.UseCloudflareEstafetteExtension || !p.DNSParams.UseExternalDNS {
+		p.DNSParams.UseExternalDNS = true
```

Comment:
Should `UseCloudflareEstafetteExtension` be default? I'm assuming current pipelines using Apigee might break if not ignore this comment

Reply:
Good thing that you spotted that one. In order to keep the backwards compatibility, both of the annotations need to be present by default and in later stages, we can introduce the breaking change where the external-dns will become the default option. 

You were referring to this, right?

Reply:
Not exactly, I was thinking what will happen to pipelines that don't change anything. They will automatically will be migrated to External DNS with this change so I thought there DNS might break

Reply:
Ok, I see. First iteration, introduce this option and let people out-in and the next iteration, make this as the default. Let me do that.

Reply:
I have changed the PR (title as well) to represent the changes.

Reply:
thanks 👍